app/migrations/20250312125210-adds_project_admin_table.cjs

```diff
@@ -0,0 +1,95 @@
+"use strict";
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.createTable("ProjectAdmin", {
+      project_admin_id: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        primaryKey: true,
+      },
+      project_id: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        references: {
+          model: "Project",
+          key: "project_id",
+        },
+        onUpdate: "CASCADE",
+        onDelete: "CASCADE",
+      },
+      user_id: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        references: {
+          model: "User",
+          key: "user_id",
+        },
+        onUpdate: "CASCADE",
+        onDelete: "CASCADE",
+      },
+      role: {
+        type: Sequelize.ENUM("ADMIN"),
```

Comment:
Should we have a default role/ another role here as well or allow setting this to null?